shared/features/_translations/features.translate.ts

```diff
@@ -11,6 +12,7 @@ export const enFeaturesTranslations = {
     notifications: enNotifications,
     socialLink: enSocialLinkTranslate,
     activityGraph: enActivityGraphTranslate,
+    cardContributionKanban: enCardContributionKanban,
```

Comment:
There is already a translation file in my PR, I don't know if you want to merge these after ?